compiler/src/dotty/tools/dotc/util/Signatures.scala

```diff
@@ -40,6 +49,18 @@ object Signatures {
     def show: String = if name.nonEmpty then s"$name: $tpe" else tpe
   }
 
+  /**
+   * Extract (current parameter index, function index, functions) method call for given position.
+   *
+   * @param pos Position for which call should be returned
+   * @return A triple containing the index of the parameter being edited, the index of functeon
+   *         being called, the list of overloads of this function).
+   */
+  def signatureHelp(pos: SourcePosition)(using Context): (Int, Int, List[Signature]) = {
```

Comment:
Could we keep the option with Path as parameter? Not sure why but `compilationUnits` are empty for me:

https://github.com/scalameta/metals/compare/main...tgodzik:scala3-signature-compat?expand=1

Unless you are able to find what I am doing wrong?